admin-dev/themes/default/template/controllers/cart_rules/form.js

```diff
@@ -333,14 +333,14 @@ $('.tab-row.active').removeClass('active');
 $(`#cart_rule_${currentFormTab}`).show();
 $(`#cart_rule_link_${currentFormTab}`).parent().addClass('active');
 
-const date = new Date();
-let hours = date.getHours();
+var date = new Date();
```

Comment:
CI is still running as I write this, but I don't think eslint will tolerate `var`, you could go with `global` or `window` to fix this resignation issue (same for other places where you used `var` )

```suggestion
window.date = new Date();
```

Reply:
Seems like eslint CI was skipped....  I had to fix something similar, for the same reassignment issue, you can see the conversation here:

https://github.com/PrestaShop/PrestaShop/pull/24356#discussion_r629205063

But since ESlint is skipped, I guess it's up to you 👍